docs/user/software/third-party/snap.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Snap
+summary: Working with Snaps
+---
+
+# Snap
+:::warning
+Note that Snaps are going to be [removed from Solus in the future][2],
+and are not fully confined in Solus with kernels newer than 6.9.
+See the section on confinement below.
+:::
+
+[Snaps][1] are a way of installing third-party packages.
+The `snapd` package installs the CLI client that can be used to install and run snaps.
+
+## Confinement warning
+
+Snaps are usually protected using `strict` confinement,
+this ensures that Snaps can't access more of your system than needed.
+The `snap` command provided by Solus shows a warning when strict confinement is not available.
+For example:
+
+> WARNING: snap is running with partial confinement. See https://help.getsol.us/docs/user/software/third-party/snap for details
+
+We recommend [migrating to Flatpak](#migrating-to-flatpak) when possible.
+The warning can be silenced by running `sudo snap hide-confinement-warning`.
+
+## Migrating to Flatpak
+
+The `unsnap` package can be used to migrate from Snap to Flatpak.
+While it does not actively remove user data in `~/snap` or `/var/lib/snapd`,
+we cannot guarantee that no data gets removed on accident.
+Please ensure you have system backups that include snap data.
```

Comment:
When referring to Snaps, is the S always capitalized? I guess the proper question is, is it a noun or a proper noun?

Reply:
Based on snapcraft.io it isn't. Updated!